server/db/models/user.js

```diff
@@ -30,6 +34,7 @@ module.exports = function (db) {
                 return _.omit(this.toJSON(), ['password', 'salt']);
             },
             correctPassword: function (candidatePassword) {
+                console.log(typeof candidatePassword);
```

Comment:
@elizabethkim828 I just put this up right after i got the kit adding working, not for merge review. I'm going to remove everything now.